src/members/Member.css

```diff
@@ -1,50 +1,57 @@
 .container-members {
-    background: #f7f9fa;
-    height: auto;
-    width: 90vw;
-    margin: 5em auto;
-    -webkit-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
-    -moz-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
-    box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
-  }
-  
+  background: #f7f9fa;
+  height: auto;
+  width: 90vw;
+  margin: 5em auto;
+  -webkit-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
+  -moz-box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
+  box-shadow: 5px 5px 5px 0px rgba(0, 0, 0, 0.4);
+}
+
 label {
-    color: #24b9b6;
-    font-size: 1.2em;
-    font-weight: 400;
-  }
-  
-  h1 {
-    color: #24b9b6;
-    padding-top: 0.5em;
-  }
-  
-  .error {
-    color: red;
-  }
-  .error-message {
-    color: #ff6565;
-    padding: 0.5em 0.2em;
-    height: 1em;
-    position: absolute;
-    font-size: 0.8em;
-  }
-  
-  .field {
-    border: 1px solid #1c6ea4;
-    width: 100%;
-    height: 2em;
-  }
-  
-  .button-group {
-    width: 100%;
-  }
-  
-  .membersList {
-    width: 85vw;
-    text-align: center;
-    padding: 2em;
-    margin: 3em 3em;
-    padding: 2em;
-  }
-  
\ No newline at end of file
+  color: #24b9b6;
+  font-size: 1.2em;
+  font-weight: 400;
+}
+
+h1 {
+  color: #24b9b6;
+  padding-top: 0.5em;
+}
+
+.error {
+  color: red;
+}
+.error-message {
+  color: #ff6565;
+  padding: 0.5em 0.2em;
+  height: 1em;
+  position: absolute;
+  font-size: 0.8em;
+}
+
+.field {
+  border: 1px solid #1c6ea4;
+  width: 100%;
+  height: 2em;
+}
+
+.button-group {
+  width: 100%;
+}
+
+.membersList {
+  width: 85vw;
+  text-align: center;
+  padding: 2em;
+  margin: 3em 3em;
+  padding: 2em;
+}
+
+@media only screen and (max-width: 644px) {
+  .container-members {
+    overflow: hidden;
+    overflow-x: scroll;
+  }
+}
+
```

Comment:
Why this extra line
There should be only one blank line at end of file and that too without any whitespaces

Reply:
@devkapilbansal , I'll do that 👍

Reply:
> @devkapilbansal , I'll do that 👍 

Also fyi we have an `.editorconfig` file in this repo, you can use it by installing the editorconfig plugin for your code editor. It will take care of these small things in the future.

Reply:
> > @devkapilbansal , I'll do that 👍
> 
> Also fyi we have an `.editorconfig` file in this repo, you can use it by installing the editorconfig plugin for your code editor. It will take care of these small things in the future.

thanks @jalajcodes 👍